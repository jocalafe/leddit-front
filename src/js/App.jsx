import React, { Component } from 'react';
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';

export default class App extends Component {
  render() {
    return (
      <MuiThemeProvider>
        <div className='app'>
          <h1> Leddit! </h1>
        </div>
      </MuiThemeProvider>
    );
  }
}
