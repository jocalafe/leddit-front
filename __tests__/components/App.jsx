import React from 'react';
import renderer from 'react-test-renderer';

import App from '../../src/js/App';

jest.mock('react-dom');

describe('App', () => {
  it('renders', () => {
    const tree = renderer.create(
      <App />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
