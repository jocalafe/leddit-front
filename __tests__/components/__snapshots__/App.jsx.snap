// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App renders 1`] = `
<div
  className="app"
>
  <h1>
     Leddit! 
  </h1>
</div>
`;
